README.md

```diff
@@ -874,7 +874,7 @@ _For a more comprehensive/advanced/better categorized/... list of Linux audio so
 - [![Open-Source Software][oss icon]](https://sourceforge.net/projects/cinepaint/) [Cinepaint](http://cinepaint.org/) - Open source deep paint software.
 - [![Open-Source Software][oss icon]](https://github.com/maoschanz/drawing) [Drawing](https://maoschanz.github.io/drawing/) - This free basic raster image editor is similar to Microsoft Paint, but aiming at the GNOME desktop.
 - [![Open-Source Software][oss icon]](https://github.com/lettier/gifcurry) [Gifcurry](https://lettier.github.io/gifcurry/) - Your open source video to GIF maker built with Haskell.
-- [Gravit](https://www.designer.io/) - Gravit Designer is a full featured free vector design app right at your fingertip.
```

Comment:
Any reason to remove this one?

If site is broken, commit message should mention it.

Or it's a conflict resolution issue?

Reply:
Reason was: It was renamed and is not called Gravit anymore and it seems to be a web service now instead of anything related to Linux, right?

I wrote this in the commit [b946b28](https://github.com/luong-komorebi/Awesome-Linux-Software/pull/814/commits/b946b28212622c576fc2b63148d6ad4396895091):
> [removed Gravit, because it was renamed and seems to be a web service now](https://github.com/luong-komorebi/Awesome-Linux-Software/pull/814/commits/b946b28212622c576fc2b63148d6ad4396895091)